docs/config.json

```diff
@@ -517,8 +517,34 @@
           "slug": "/desktop-access/getting-started/"
         },
         {
-          "title": "Configuration",
-          "slug": "/desktop-access/reference/configuration/"
+          "title": "Access Controls",
+          "slug": "/desktop-access/rbac/"
+        },
+        {
+          "title": "Reference",
+          "slug": "/desktop-access/reference/",
```

Comment:
This link 404s. I think we should either add a reference.mdx page with a tile for each child page or use the slug for one of the child pages instead.